Stellar-operation-bind-external-system-id.x

```diff
@@ -11,10 +11,14 @@ Result: BindExternalSystemAccountIdResult
 
 */
 
+//: BindExternalSystemAccountIdOp is used to bind source account to
+//: external system account ID pool
```

Comment:
... to bind a particular account to the external system account represented by account ID taken from pool